portfolio/src/main/webapp/sheetsApi.js

```diff
@@ -0,0 +1,146 @@
+// Client ID and API key from the Developer Console
+var CLIENT_ID =
+    '499747085593-hvi6n4kdrbbfvcuo1c9a9tu9oaf62cr2.apps.googleusercontent.com';
+var API_KEY = '';
+
+// Array of API discovery doc URLs for APIs used by the quickstart
+var DISCOVERY_DOCS =
+    ['https://sheets.googleapis.com/$discovery/rest?version=v4'];
+
+// Authorization scopes required by the API; multiple scopes can be
+// included, separated by spaces.
+var SCOPE_READ_WRITE = 'https://www.googleapis.com/auth/spreadsheets';
+// Error codes that the process can throw
```

Comment:
Consider using object for different error codes.

var ERROR_CODE = {
  ...: ...
  ...: ...
};

Reply:
Done.